### PR TITLE
made export function, brushed up chart, corrected tooltip error

### PR DESCRIPTION
**Pull request by jorntx**
Made client side export to CSV function
Brushed up the looks of the chart
Corrected wrong data in tooltips
Added about text

### CODE REVIEW

**Comment by jorntx** [comment]
@jesperlauridsen needs approval before merge - much of the code is already reviewed by you in Solrwayback

**Review by jesperlauridsen — APPROVED** [review]
Looks very noice!

**Pull request merged**
